src/main/java/uk/gov/hmcts/reform/sandl/snlevents/service/StatusServiceManager.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.sandl.snlevents.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sandl.snlevents.model.db.Statusable;
+
+@Service
+public class StatusServiceManager {
```

Comment:
Does this service make sense?

Reply:
If later on there will be more complicated methods inside